packages/desktop-client/src/components/reports/ChooseGraph.tsx

```diff
@@ -25,6 +25,7 @@ type ChooseGraphProps = {
   scrollWidth: number;
   setScrollWidth: (value: number) => void;
   months: Month[];
+  OnChangeLegend;
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏we should follow `camelCase` for props. And also we're missing a type here.

```suggestion
  onChangeLegend: (something: something) => something;
```